--- bad-behavior/bad-behavior-core.php~	2005-12-02 18:48:13.000000000 +0200
+++ bad-behavior/bad-behavior-core.php	2006-07-10 15:58:20.692823726 +0300
@@ -5,7 +5,7 @@
 
 // die("Do NOT check Bad Behavior out of svn! Install the <a href=\"http://www.ioerror.us/software/bad-behavior/\">current version</a>.");
 
-define(WP_BB_VERSION, "1.2.4");
+define('WP_BB_VERSION', "1.2.4");
 
 require_once(WP_BB_CWD . "/bad-behavior-functions.php");
 
